### PR TITLE
Style Overhaul with Improvements for Mobile

### PR DESCRIPTION
**Pull request by hyjk2000**
- Breath some fresh air of CSS Grid to our overall page layout
- Minor tweaks to mobile view for aesthetics / usability
- Fix visual bugs in the meantime

### CODE REVIEW

File: src/components/Actions/Actions.css [hunk 1]
```diff
@@ -69,8 +63,18 @@
     color: white;
   }
 
+  .operations {
+    flex-grow: 1;
+    justify-content: flex-end;
+  }
+
+  .operations li {
+    &:not(:first-child) {
```
Comment (just4fun) [inline src/components/Actions/Actions.css]: There is only one `li` element inside `ul.operations`, it seems like this rule will never be applied.

File: src/components/Actions/Actions.css [hunk 2]
```diff
@@ -5,22 +5,15 @@
 
   display: flex;
   flex-wrap: wrap;
-  justify-content: space-around;
```
Comment (just4fun) [inline src/components/Actions/Actions.css]: `ul.scores` will not be centered horizontally without this line in iPad portrait mode.

File: src/components/Actions/Actions.css [hunk 3]
```diff
@@ -69,8 +63,18 @@
     color: white;
   }
 
+  .operations {
+    flex-grow: 1;
+    justify-content: flex-end;
```
Comment (just4fun) [inline src/components/Actions/Actions.css]: Operation button will still be located in right of screen in iPad portrait mode.

File: src/pages/Landing/Landing.css [hunk 4]
```diff
@@ -2,14 +2,15 @@
...
-  left: 0;
+  display: grid;
+  grid-template-rows: 1fr auto 1fr 3rem;
+  height: 100vh;
+  background-size: cover;
+  /* background-position: center; */
 
   .Form {
-    margin: 30vh auto;
+    -ms-grid-row: 2;
```
Comment (just4fun) [inline src/pages/Landing/Landing.css]: Is this in use?

File: src/pages/Room/Room.css [hunk 5]
```diff
@@ -1,5 +1,9 @@
 .Room {
-  position: relative;
+  display: grid;
+  grid-template-rows: repeat(4, min-content);
+  grid-row-gap: 1rem;
+  row-gap: 1rem;
```
Comment (just4fun) [inline src/pages/Room/Room.css]: Are `grid-row-gap` and `row-gap` the same?

**Comment by just4fun** [comment]
Looks almost good to me, just have an small UI issue in iPad portrait mode and some questions about CSS grid.

BTW, I like CSS grid and the animation approach with `react-transition-group`.

File: src/components/Actions/Actions.css (hunk 1 above)
Comment (hyjk2000) [reply src/components/Actions/Actions.css]: `Clear` button was there alongside `Show` button in the original design. I'm still thinking about bringing it back so I kept the necessary style.

File: src/components/Actions/Actions.css (hunk 2 above)
Comment (hyjk2000) [reply src/components/Actions/Actions.css]: I overlooked this, it's fixed now.

File: src/pages/Room/Room.css (hunk 5 above)
Comment (hyjk2000) [reply src/pages/Room/Room.css]: Good question. `grid-row-gap` is deprecated in favor of prefix-less `row-gap`, but still needed for some browsers which does not implement `row-gap` yet such as Safari and Samsung Internet. I'm not sure if autoprefixer handles this, we can remove `grid-row-gap` if so.

File: src/pages/Landing/Landing.css (hunk 4 above)
Comment (hyjk2000) [reply src/pages/Landing/Landing.css]: Yes, it is. Grid prefixes are [disabled by default](https://github.com/postcss/autoprefixer/issues/817) in Autoprefixer.

Comment (hyjk2000) [reply src/pages/Landing/Landing.css]: Just noticed that Edge 16 support the new specification. I'll remove this.

**Comment by just4fun** [comment]
tenor-121429946

**Pull request merged**